clang-tools-extra/clangd/unittests/DiagnosticsTests.cpp

```diff
@@ -898,6 +898,34 @@ TEST(DiagnosticTest, ClangTidySelfContainedDiags) {
                 withFix(equalToFix(ExpectedDFix))))));
 }
 
+TEST(DiagnosticTest, ClangTidySelfContainedDiagsFormatting) {
```

Comment:
Tests for clang tidy should be added here: clang-tidy/checkers/modernize/use-override.cpp

Reply:
Added a case in clang-tools-extra/test/clang-tidy/checkers/modernize/use-override.cpp.